pkg/workloadmeta/README.md

```diff
@@ -0,0 +1,32 @@
+# Workloadmeta Store
```

Comment:
It might be worth to mention somewhere in this doc that we have telemetry for the workloadmeta store: https://github.com/DataDog/datadog-agent/blob/main/pkg/workloadmeta/telemetry/telemetry.go

Reply:
We also have the `agent workload-list` command. It's useful for debugging. It might be good to mention it somewhere.